internal/conf/zookeeper_test.go

```diff
@@ -0,0 +1,42 @@
+package conf
+
+import (
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/yaml.v3"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+)
+
+func ZkReadConfig(t *testing.T) *Config {
+	wd, _ := os.Getwd()
```

Comment:
In fact you don't have a function called `ReadConfig`, so this shouldn't be a unit test. If this is a utility function you can:
```golang
func readConfig() *Config {
    ....
}
```